### PR TITLE
fix YAML parsing errors

### PR DESCRIPTION
**Pull request by tilleryd**
@justincely and @joshua-rutherford, you both reported errors with YAML parsing. This should fix both cases.

Please re-deploy when you test this to confirm the fixes.

### CODE REVIEW

File: greymatter.yaml [hunk 1]
```diff
@@ -554,7 +554,6 @@ jwt:
         key: jwt.key.pem
 
   redis:
-    openshift:
```
Comment (joshua-rutherford) [inline greymatter.yaml]: So this is only an issue on 2.14.0 not the documented 2.14.3 version of helm.  I don't know if we need to change this.

Comment (joshua-rutherford) [reply greymatter.yaml]: But if we do we can change this in postgres too.

File: docs/Service Accounts.md [hunk 2]
```diff
@@ -30,7 +30,13 @@ The following list gives the service that needs a service account along with the
...
 - `spire-server` - spire subchart - `.Values.spire.server.serviceAccount`
 
-If you're deploying into an environment where Helm/Tiller doesn't have sufficient permissions to create service accounts, you'll need to apply the [greymatter-service-accounts.yaml](../greymatter-service-accounts.yaml) file first. Be sure to set the appropriate namespace you're deploying to throughout that file. The user to apply this file will need to have sufficient permissions to create accounts. The custom values file will also need to be updated to ensure Helm doesn't try to create the accounts.
+If you're deploying into an environment where Helm/Tiller doesn't have sufficient permissions to create service accounts, you'll need to apply the [greymatter-service-accounts.yaml](../greymatter-service-accounts.yaml) file first. Be sure to set the appropriate namespace you're deploying to throughout that file. The user to apply this file will need to have sufficient permissions to create accounts.
+
+```sh
+oc apply -f greymatter-service-accounts.yaml
+```
+
+The custom values file will also need to be updated to ensure Helm doesn't try to create the accounts.
```
Comment (joshua-rutherford) [inline docs/Service Accounts.md]: This seems really vague.  Can we:

a. provide directions on what to remove.
b. provide a version of the custom config that does this.

File: greymatter.yaml (hunk 1 above)
Comment (tilleryd) [reply greymatter.yaml]: I'm ok with the change because now it's valid YAML.

File: docs/Service Accounts.md (hunk 2 above)
Comment (tilleryd) [reply docs/Service Accounts.md]: Yes, I'll address this.

Comment (tilleryd) [reply docs/Service Accounts.md]: @joshua-rutherford Due to having 4 different custom files already in the repo I opted to clarify this all within this doc.

**Comment by tilleryd** [comment]
@joshua-rutherford this is ready for another review

File: docs/Service Accounts.md (hunk 2 above)
Comment (justincely) [reply docs/Service Accounts.md]: These instructions are going to evolve as we use it.  I'm ok with where it is now

**Pull request merged**